cmd/expiration-mailer/main_test.go

```diff
@@ -258,6 +258,42 @@ func TestFindExpiringCertificates(t *testing.T) {
 		Status: core.OCSPStatusGood,
 	}
 
+	// Expires in 2 days, already renewed 3 days ago, no nag sent
```

Comment:
This test function is already quite long. As I mentioned above, if you write an `isRenewed` function, you can test that independently, with maybe an additional test that processCerts uses `isRenewed` correctly.